### PR TITLE
Adopt new ava and mockserver

### PR DESCRIPTION
**Pull request by mdlavin**
This PR also includes the changes for https://github.com/lifeomic/lambda-tools/pull/60. When this merges, Greenkeeper should be enabled again.

The changes here adopt both Ava 1.0 and the MockServer 5.5 release. Apparently, MockServer decided to make some breaking changes in the 5.5 release and so I'm going to do a major version bump of this project to highlight the required change. I'll also create release notes in GitHub for the change.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@lifeomic/lambda-tools",
-  "version": "6.0.1",
+  "version": "7.0.0-beta",
```
Comment (mdlavin) [inline package.json]: This is a placeholder so make sure that no 6.x versions are released with an 5.5 MockServer library in them

File: src/mockServerLambda.js [hunk 2]
```diff
@@ -33,5 +33,5 @@ exports.verifyInvocation = (mockServerClient, functionName, requestBody, times)
       json: JSON.stringify(requestBody),
       matchType: 'ONLY_MATCHING_FIELDS'
     }
-  }, times, true);
+  }, times, times);
```
Comment (mdlavin) [inline src/mockServerLambda.js]: Here is the primary breaking MockServer API change

Comment (Druotic) [reply src/mockServerLambda.js]: For anyone following along - here's the breaking change: https://github.com/jamesdbloom/mockserver-client-node/commit/dce5b438a845da9bb5353a430bb911c1cd12ecd8#diff-f884a33a5b565b9215e48c815f97e3eeR417 (v5.5.0) Their docs are still outdated, though: https://github.com/jamesdbloom/mockserver-client-node#verify-requests

File: test/mockServerLambda.test.js [hunk 3]
```diff
@@ -141,7 +141,15 @@ test('Lambda function invocations can be verified', async (test) => {
   // Verifying no invocations should succeed
   await verifyInvocation(mockServerClient, functionName, expectedRequestBody, 0);
   // Verifying one invocation should fail
-  await test.throws(verifyInvocation(mockServerClient, functionName, expectedRequestBody, 1), /Request not found exactly once/);
+  try {
+    await verifyInvocation(mockServerClient, functionName, expectedRequestBody, 1);
+    throw new Error('Verification should have thrown an error');
+  } catch (error) {
+    // mockClientServer throws `string` type errors instead of real Errors.
```
Comment (Druotic) [inline test/mockServerLambda.test.js]: Eww :( Nice strictness by the ava upgrade, though.

**Pull request merged**